app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala

```diff
@@ -368,12 +368,12 @@ object CompleteReturn {
   ): Option[Option[CompleteRepresenteeAnswers]] =
     (individualUserType, representeeAnswers) match {
       case (
-            Some(Capacitor) | Some(PersonalRepresentative),
+            Some(Capacitor) | Some(PersonalRepresentative) | Some(PersonalRepresentativeInPeriodOfAdmin),
```

Comment:
you could replace all of these with just `Some(RepresentativeType)` to make it shorter

Reply:
From the looks of it we want to exclude `Self`

Reply:
`RepresentativeType` does exclude `Self` - it's basically everything other than `Self`

Reply:
Oh sorry! I misread the file, that extends `IndividualUserType`. Updating now.